XMLParser/XMLParser.h

```diff
@@ -94,5 +94,5 @@ class XMLParser final : public IXMLParser
 
 private:
 	std::shared_ptr<XMLNode> m_root;
-	Logger LOG;
+	//Logger LOG;
```

Comment:
remove commented code

Reply:
This is only for demo
Logger in new branch is using separate parser